pkg/client/logs.go

```diff
@@ -150,50 +172,60 @@ func (s *SonobuoyClient) LogReader(cfg *LogConfig) (*Reader, error) {
 	if err != nil {
 		return nil, err
 	}
+	stopPodFetch := make(chan bool)
+	podListChan := make(chan v1.PodList)
+	podsFetchError := make(chan error)
+	go getPodsToStreamLogs(client, cfg, stopPodFetch, podListChan, podsFetchError)
 
-	pods, err := getPodsForLogs(client, cfg)
-	if err != nil {
-		return nil, err
-	}
-
-	// We must make sure the error channel has capacity enough so that it never blocks
-	// in case of early exits.
-	numContainers := 0
-	for _, pod := range pods.Items {
-		numContainers += len(pod.Spec.Containers)
-	}
-
-	errc := make(chan error, numContainers)
+	errc := make(chan error, 5)
```

Comment:
Why 5?

Reply:
I was just trying something out. This shouldn't have made it into the commit.